packages/netlify-cms-core/src/constants/configSchema.js

```diff
@@ -96,7 +96,7 @@ const getConfigSchema = () => ({
           },
           format: { type: 'string', enum: Object.keys(formatExtensions) },
           extension: { type: 'string' },
-          frontmatter_delimiter: { type: 'string' },
+          frontmatter_delimiter: { type: ['string', 'array'] },
```

Comment:
This schema will allow an array of any type, rather than only an array of strings. It should be modified to specify that the children of arrays should be strings.

Reply:
@Benaiah good catch! Updated this in 0aa3ac4.